src/loki/status_action.cc

```diff
@@ -55,15 +57,23 @@ void loki_worker_t::status(Api& request) const {
 
   // get _some_ tile
   const static baldr::graph_tile_ptr tile = get_graphtile(reader);
+  // TODO: get this from the connectivity_map once transit is implemented there
 
   if (connectivity_map) {
     status->set_bbox(connectivity_map->to_geojson(2));
+    const bool has_transit_tiles =
+        connectivity_map->level_color_exists(TileHierarchy::GetTransitLevel().level);
+    status->set_has_transit_tiles(has_transit_tiles);
+  } else {
+    const static bool has_transit_tiles = !reader->GetTileSet(3).empty();
+    status->set_has_transit_tiles(has_transit_tiles);
```

Comment:
if we have a connectivity map, great, we'll use the newly added transit level in there via a new tiny function